web/src/lib/components/admin-page/settings/admin-settings.svelte

```diff
@@ -0,0 +1,80 @@
+<svelte:options accessors />
+
+<script lang="ts">
+  import { SystemConfigDto, api } from '@api';
+  import {
+    notificationController,
+    NotificationType,
+  } from '$lib/components/shared-components/notification/notification';
+  import { handleError } from '$lib/utils/handle-error';
+  import _ from 'lodash';
+  import type { SettingsEventType } from './admin-settings';
+  import { createEventDispatcher, onMount } from 'svelte';
+
+  export let config: SystemConfigDto;
+
+  let savedConfig: SystemConfigDto;
+  let defaultConfig: SystemConfigDto;
```

Comment:
In a future PR we may want to never refresh this, since it won't ever change. In that case, in can be passed into the component from the parent page, which can load it once.